odd-platform-api/src/main/java/com/provectus/oddplatform/auth/CognitoOidcLogoutSuccessHandler.java

```diff
@@ -0,0 +1,52 @@
+package com.provectus.oddplatform.auth;
+
+import java.net.URI;
+import java.nio.charset.StandardCharsets;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.server.reactive.ServerHttpResponse;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.server.WebFilterExchange;
+import org.springframework.security.web.server.authentication.logout.ServerLogoutSuccessHandler;
+import org.springframework.security.web.util.UrlUtils;
+import org.springframework.web.server.WebSession;
+import org.springframework.web.util.UriComponents;
+import org.springframework.web.util.UriComponentsBuilder;
+import reactor.core.publisher.Mono;
+
+@RequiredArgsConstructor
+public class CognitoOidcLogoutSuccessHandler implements ServerLogoutSuccessHandler {
+
+    private final String logoutUrl;
+    private final String clientId;
+
+    @Override
+    public Mono<Void> onLogoutSuccess(WebFilterExchange exchange, Authentication authentication) {
+        ServerHttpResponse response = exchange.getExchange().getResponse();
+        response.setStatusCode(HttpStatus.FOUND);
+
+        final var requestUri = exchange.getExchange().getRequest().getURI();
+
+        final var fullUrl = UrlUtils.buildFullRequestUrl(requestUri.getScheme(),
+            requestUri.getHost(), requestUri.getPort(),
+            requestUri.getPath(), requestUri.getQuery());
+
+        UriComponents baseUrl = UriComponentsBuilder
```

Comment:
Please use finals. Let's add it to the linter rules if possible

Reply:
I've put them in this class, but I'll edit linter in a separate PR.